### PR TITLE
Added new translation files

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

Added new translation files. @ydrain can you please confirm these are finalised.

/cc @zendesk/quokka

### References
 - Jira link: n/a

### Risks
 - None

### CODE REVIEW

**Comment by princemaple** [comment]
:+1:

File: translations/zh-cn.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "button": {
+    "cancel": "取消",
+    "imageInput": "添加图像链接",
+    "sendMessage": "广播",
```
Comment (princemaple) [inline translations/zh-cn.json]: this is a bit weird on a button though.

Comment (ydrain) [inline translations/zh-cn.json]: @princemaple About the translation for the "Broadcast" button? It's a hard word to translate, and I was expected to have some translation issues when I saw the English string. What translation do you suggest?

cc: @mariasica

**Comment by ydrain** [comment]
:+1: 

We will follow up the zh-cn translation with @princemaple and also perform LQA.

File: translations/zh-cn.json (hunk 1 above)
Comment (princemaple) [inline translations/zh-cn.json]: @ydrain I don't have good ideas either. If it's allowed / possible, I'd just translate it as if it was "send".

**Pull request merged**